WordPress/Classes/ViewRelated/NUX/LoginProloguePageViewController.swift

```diff
@@ -0,0 +1,107 @@
+import UIKit
+
+class LoginProloguePageViewController: UIPageViewController {
+    var pages: [UIViewController] = []
+    fileprivate var pageControl: UIPageControl?
+    fileprivate var bgAnimation: UIViewPropertyAnimator?
+    fileprivate struct Constants {
+        static let topPadding: CGFloat = 20.0
+        static let pageControlHeight: CGFloat = 40.0
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        dataSource = self
+        delegate = self
+
+        for ii in 1...5 {
+            if let page = storyboard?.instantiateViewController(withIdentifier: "promo\(ii)") {
+                pages.append(page)
+            }
+        }
+        setViewControllers([pages[0]], direction: .forward, animated: false)
+        view.backgroundColor = backgroundColor(for: 0)
+
+        addPageControl()
+    }
+
+    func addPageControl() {
+        let newControl = UIPageControl()
+
+        view.addSubview(newControl)
```

Comment:
Anchor assignments have to go later so that they can be satisfied, but `translatesAutoresizingMaskIntoConstraints` can absolutely be moved 👍